internal/ask/form/submission/submission.go

```diff
@@ -107,7 +107,7 @@ type Answer struct {
 	Identity     bool        `json:"identity" bson:"identity"`
 	Answer       interface{} `json:"answer" bson:"answer"`
```

Comment:
Use the of empty interface needs to be reviewed. We don't want to write generic or generalized code if not necessary or it if is not making the code simpler to write or reason about. So just ask the question.